### PR TITLE
Apply project settings to o.e.equinox.security.UI

### PR DESCRIPTION
**Pull request by Torbjorn-Svensson**
1. Convert to LF and ensure LF at EOF:
```shell
find . -type f -print0 | \
  xargs -r0 perl -le 'for (@argv) { print if -f && -T }' | \
  xargs -rd'\n' dos2unix -e
```

2. Format java files:
```shell
eclipse -consolelog -nosplash -application org.eclipse.jdt.core.JavaCodeFormatter \
  -config .settings/org.eclipse.jdt.core.prefs . -data `mktemp -d`
```

### CODE REVIEW

File: bundles/org.eclipse.equinox.security.ui/src/org/eclipse/equinox/internal/security/ui/storage/TabContents.java [hunk 1]
```diff
@@ -94,17 +95,14 @@ public TabContents(TabFolder folder, int index, final Shell shell) {
...
-		setButtonSize(buttonExport, minButtonWidth);
-		buttonExport.addSelectionListener(new SelectionAdapter() {
-			public void widgetSelected(SelectionEvent e) {
-				export();
-			}
-		});
-		*/
+		/*
+		 * Removed for the time being. In future modify/show/export operations could be
+		 * re-introduced with some special access token required to be entered by the
```
Comment (akurtakov) [inline bundles/org.eclipse.equinox.security.ui/src/org/eclipse/equinox/internal/security/ui/storage/TabContents.java:100]: Broken code snippet please wrap in @code .

Comment (Torbjorn-Svensson) [reply bundles/org.eclipse.equinox.security.ui/src/org/eclipse/equinox/internal/security/ui/storage/TabContents.java]: This is a regular java comment, not a javadoc comment, so `@code` would not make a difference here.
The only way I managed to get this preserved was using `@formatter:off` and `@formatter:on`. Is this okay?

**Review by akurtakov — APPROVED** [review]

**Comment by Torbjorn-Svensson** [comment]
@akurtakov Are the build error (API check failed) a real error or due to some tycho/maven update? I think this pr did not have any API problems before the force-push today.

**Comment by akurtakov** [comment]
Probably related to https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/pull/1437

**Comment by laeubi** [comment]
I'll replay the build to see if we can get some more details.

**Comment by iloveeclipse** [comment]
> Probably related to [eclipse-platform/eclipse.platform.releng.aggregator#1437](https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/pull/1437)

See discussion on https://github.com/eclipse-pde/eclipse.pde/issues/782

**Pull request merged**